### PR TITLE
Small typo and formatting fixes for docs

### PR DESCRIPTION
**Pull request by alexwlchan**
List of changes:

* Add quotes around the title of the final question on the FAQs page to bring
  it in line with the style of the other titles on the page.
* Fix formatting of `eth0` in `fuel-integration.rst`.
* Fix list formatting in `opens-chef-install.rst`.
* Add missing text to `redhat-opens-install.rst`.
* Fix indentation in `redhat-opens-install.rst`.
* Fix indentation in `ubuntu-opens-install.rst`.
* Tidy up a couple of links.
* Replace hyphens with en dashes in a few places.

I tested all of these changes with a local build of the HTML docs.

### CODE REVIEW

File: docs/source/fuel-integration.rst [hunk 1]
```diff
@@ -290,7 +290,7 @@ Detailed Observations
 
 This section records some more detailed notes about the state of the
 cluster that results from following the above procedure with HEAD
-commit 854d2353 from https://github.com/Metaswitch/fuel-library.
+commit 854d2353 from `our fork of the Fuel library <https://github.com/Metaswitch/fuel-library>`__.
```
Comment (Lukasa) [inline docs/source/fuel-integration.rst]: Can you wrap this line please?

**Comment by Lukasa** [comment]
ok to test

**Comment by Lukasa** [comment]
One small note, @alexwlchan!

File: docs/source/fuel-integration.rst (hunk 1 above)
Comment (alexwlchan) [inline docs/source/fuel-integration.rst]: Fixed.

**Pull request merged**